build.rs

```diff
@@ -31,6 +31,7 @@ fn create_util_map() -> HashMap<&'static str, &'static str> {
     hashmap.insert("head", "posix");
     hashmap.insert("sh", "posix");
     hashmap.insert("sleep", "posix");
+    hashmap.insert("nc", "networking");
```

Comment:
I think this line should be put with `ping` (which is in the "networking section).

Reply:
done